executor/distsql.go

```diff
@@ -888,6 +891,7 @@ func (e *XSelectTableExec) Next() (*Row, error) {
 		}
 		if rowData == nil {
 			// Finish the current partial result and get the next one.
+			e.partialResult.Close()
```

Comment:
1. partialResult.Close() is a null operation, in distsql/distsql.go

```
// Close closes the sub result.
func (pr *partialResult) Close() error {
	return nil
}
```

2. I'm afraid Close is called many times.